.travis.yml

```diff
@@ -4,8 +4,11 @@ rvm:
   - 2.0.0
   - 2.1.0
   - 2.2.0
+  - 2.3.0
+  - 2.4.0
+  - 2.5.0
 # Bundler 1.13 added support for `ruby RUBY_VERSION` in the Gemfile
-before_install: gem install bundler -v '>= 1.13.0'
+before_install: gem install bundler -v '1.17.3'
```

Comment:
I set this version because it's the last version to allow use ruby v.1.9.3 Without this fix it's possible that Travis will use bundler wich not allowed ruby less than 2.3.0